cypress/support/fragments/finance/funds/funds.js

```diff
@@ -146,6 +146,19 @@ export default {
     ]);
   },
 
+  transfer: (thisFund, fromFund) => {
+    cy.do([
+      Button('Actions').click(),
+      Button('Transfer').click(),
+      Button({ name: 'toFundId' }).click(),
+      SelectionOption(`${thisFund.name} (${thisFund.code})`).click(),
+      Button({ name: 'fromFundId' }).click(),
+      SelectionOption(`${fromFund.name} (${fromFund.code})`).click(),
+      TextField({ name: 'amount' }).fillIn('10'),
+      Modal({ id: 'add-transfer-modal' }).find(Button('Confirm')).click(),
```

Comment:
Button('Confirm') move to const variable

Reply:
Done